templates/web/base/admin/body-form.html

```diff
@@ -75,8 +75,7 @@
     </div>
     <p>
         <label for="area_ids">[% loc('Area covered') %]</label>
-        <select class="form-control" name="area_ids" id="area_ids" multiple>
-            <option value=""> -- [% loc('Select an area') %] -- </option>
+        <select class="form-control js-multiple" name="area_ids" id="area_ids" multiple data-none="-- [% loc('Select an area') %] --">
```

Comment:
Do we want to keep the '--' here, do you think?

Reply:
I left it in to match the other `<select>` elements in the same form.